### PR TITLE
add elysia, stricjs and hono

### PR DESCRIPTION
**Pull request by jim-king-2000**
Add bun frameworks of elysia, stricjs and hono to the benchmark.

### CODE REVIEW

**Comment by waghanza** [comment]
@SaltyAom we plan to add `elysia` in here. Are you not opposed on such a thing ?

**Comment by waghanza** [comment]
@Yusuke we plan to introduce `hono` in here. Are you opposed in such a thing?

**Comment by waghanza** [comment]
@bunsvr we are going to introduce `stric` in here. Are you opposed on such a thing?

File: javascript/elysia/config.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+framework:
+  website: elysiajs.com
+  version: 0.5.17
```
Comment (waghanza) [inline javascript/elysia/config.yaml:3]:
```suggestion
  version: 0.5
```

File: javascript/elysia/package.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "elysia": "^0.5.17"
```
Comment (waghanza) [inline javascript/elysia/package.json:3]:
```suggestion
    "elysia": "~0.5.17"
```

File: javascript/hono/config.yaml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+framework:
+  website: hono.dev
+  version: 3.2.5
```
Comment (waghanza) [inline javascript/hono/config.yaml:3]:
```suggestion
  version: 3.2
```

File: javascript/hono/package.json [hunk 4]
```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "hono": "^3.2.5"
```
Comment (waghanza) [inline javascript/hono/package.json:3]:
```suggestion
    "hono": "~3.2.5"
```

File: javascript/stricjs/config.yaml [hunk 5]
```diff
@@ -0,0 +1,6 @@
+framework:
+  website: stricjs.gitbook.io/docs/quick-start/intro
+  version: 3.0.2
```
Comment (waghanza) [inline javascript/stricjs/config.yaml:3]:
```suggestion
  version: 3.0
```

File: javascript/stricjs/package.json [hunk 6]
```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "@stricjs/router": "^3.0.2"
```
Comment (waghanza) [inline javascript/stricjs/package.json:3]:
```suggestion
    "@stricjs/router": "~3.0.2"
```

**Review by waghanza — CHANGES_REQUESTED** [review]
Thanks for introducing `bun` @jim-king-2000. You will need to add
```yml
bun:
  command: bun run app.ts
```
in `javascript/config.yaml` and create a simple dockerfile as `javascript/bun.Dockerfile`
```dockerfile
FROM oven/bun:0.6

WORKDIR /usr/src/app

{{#files}}
  COPY '{{source}}' '{{target}}'
{{/files}}

RUN apt-get -qq update

{{#deps}}
  RUN apt-get -qy install {{{.}}}
{{/deps}}

{{#bootstrap}}
  RUN {{{.}}}
{{/bootstrap}}

{{#environment}}
  ENV {{{.}}}
{{/environment}}

{{#fixes}}
  RUN {{{.}}}
{{/fixes}}

CMD {{{command}}}
```

**Comment by jim-king-2000** [comment]
Update the pull request.

**Comment by waghanza** [comment]
Thanks @jim-king-2000, let's way a bit before merge (if there is no opposition for any framework in this PR)

**Comment by jim-king-2000** [comment]
Looking forward to the benchmark result. :blush:

**Comment by waghanza** [comment]
I'll include those project on next run (monday)

**Review by waghanza — APPROVED** [review]

**Comment by jim-king-2000** [comment]
Glad to see it has been merged. I could add more frameworks. May I send the PR directly in the future?

**Comment by waghanza** [comment]
Sure. You can propose any framework you want, but please ping the author / core members to see if it is valuable to include it here

**Comment by waghanza** [comment]
Those frameworks has been included https://web-frameworks-benchmark.netlify.app/result?f=hono,stricjs,elysia

**Pull request merged**